src/inverseRobot.js

```diff
@@ -7,7 +7,19 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  for (const oldKey in robot) {
+    const value = oldKey;
+    const newKey = robot[value];
+
+    if (robot.hasOwnProperty(newKey)) {
+      return null;
+    }
+
+    robot[newKey] = value;
+    delete robot[oldKey];
+  }
+
+  return robot;
```

Comment:
We have task "Create a 'inverseRobot' function that takes 'robot' as a parameter and **returns a new object** in which keys will change places with values."

You`ve returned old object.